VisualizationBase/src/items/VInfoNode.cpp

```diff
@@ -49,7 +52,12 @@ void VInfoNode::initializeForms()
 void VInfoNode::determineChildren()
 {
 	Super::determineChildren();
-	browser_->setContent(node()->infoHtml());
+	if (node()->hasContentChanged())
+	{
+		browser_->setContent(node()->infoHtml());
+		browser_->browser()->page()->mainFrame()->addToJavaScriptWindowObject("operations",
```

Comment:
Instead of reaching deeply into the browser here, just modify the VBrowserCode to have an additional method which allows you to register your function. That way we keep the dependency on WebKit in that other plugin. so here you'd have just: `browser_->addJavascriptFunction(InfoJavascriptFunctions::instance());`